docs/shared/features/powerpack/custom-caching.md

```diff
@@ -1 +1,33 @@
 # Change the Remote Cache Storage Location
+
+The recommended way to enable the [remote cache](/ci/features/remote-cache) is to use Nx Replay and have Nx Cloud share the task cache across your organization. For those organizations that are unable to use Nx Cloud, Nx offers official plugins that are enabled by [Nx Powerpack]() to customize your remote cache location in a fast and secure manner.
+
+In Nx 20, the _local_ cache mechanism is getting faster and more secure by storing the cached metadata and project graph in a database instead of directly on the file system. The new cache is faster because it limits the fundamentally slow speed of interacting with the file system. The new cache is more secure because it reduces the amount of information that other processes with file system access could read or modify in the local cache directory. The [`@nx/powerpack-s3-cache`](/nx-api/powerpack-s3-cache) and [`@nx/powerpack-shared-fs-cache`](/nx-api/powerpack-shared-fs-cache) plugins enable you to leverage that speed and security with a custom _remote_ cache storage location.
+
+{% callout type="note" title="Custom Task Runners are Deprecated in Nx 20" %}
```

Comment:
Do you think this would be relevant to reading this feature page? I know how it is related because I'm aware of the underlying implementation, but I'd almost not link it here as someone coming in just wanting to leverage remote caching with powerpack might be confused about what this means for them.